demos/vc_issuer/tests/issue_credential.rs

```diff
@@ -732,6 +745,25 @@ fn should_fail_configure_if_not_controller() {
     assert_matches!(result, Err(e) if format!("{:?}", e).contains("Only a controller can call configure"));
 }
 
+#[test]
+fn should_set_alternative_origins() {
+    let env = env();
+    let issuer_id = install_canister(&env, VC_ISSUER_WASM.clone());
+    let alternative_origins = r#"{"alternativeOrigins":["https://test.issuer"]}"#;
+    api::set_alternative_origins(&env, issuer_id, alternative_origins).expect("API call failed");
+
+    let request = HttpRequest {
+        method: "GET".to_string(),
+        url: "/.well-known/ii-alternative-origins".to_string(),
+        headers: vec![],
+        body: ByteBuf::new(),
+        certificate_version: Some(2),
+    };
+    let http_response = http_request(&env, issuer_id, &request).expect("HTTP request failed");
```

Comment:
Could we check this before calling `set_alternative_origins` as well to test that it has indeed changed?